### DISCUSSION TITLE
What are in your opinion the most interesting projects built with Bevy?

### DISCUSSION THREAD

**Discussion by muradbu** [bevyengine/bevy#9044] [General]
I'm not a game developer, but I really do like looking up "{game_engine} projects" on Google, Twitter, YouTube, etc to see what awesome stuff people are building. I love it, especially from indie developers. With Bevy, when I lookup said query on YouTube I mostly find educational content though. Itch has some nice things.

However, I'm curious what everyone else is following and want to hear from _you_!

**Comment by nicopap** [comment]
Honestly, in truth, you are absolutely right there isn't much larger-scale / production projects using bevy.

* [Noumenal](https://noumenal.app/) production Ios app using bevy
* [Fish Folk: Jumpy](https://github.com/fishfolk/jumpy) an open source duck-game-like

Now, if you allow for smaller projects, such as bevy jam games. My favorite is:

* [Beam Machines](https://partial-insanity.itch.io/beam-machines) a Infinifactory clone I really really really like.
* [Shanty Quest](https://jabuwu.itch.io/shanty-quest) Very high production value, especially given that it was made in 10 days
* [Subfuse](https://dgriffin.itch.io/subfuse) Also an impressively high production value for a 10 days project

**Edit**: I forgot about [Tiny glade](https://store.steampowered.com/app/2198150/Tiny_Glade/) Which is built using `bevy_ecs` (though not the bevy renderer)

**Reply by muradbu** [reply]
Fish Folk looks like a fun game!

**Comment by NiklasEi** [comment]
[Emergence](https://github.com/Leafwing-Studios/Emergence) is a rather large open source project using Bevy to build an "organic factory-builder set in a micro-scale, post-apocalyptic future".

[Tunnet](https://store.steampowered.com/app/2286390/Tunnet/) is an upcoming game on steam about a network technician in an underground facility.

Generally, the #showcase channel on the Bevy discord is the place to watch. Lots of very nice projects in the works.

**Reply by NiklasEi** [reply]
https://discord.com/channels/691052431525675048/692648638823923732/1109618154734370896 is one of the most exiting game projects in my opinion.
